app/test_main.py

```diff
@@ -1,3 +1,43 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age_list",
+    [
+        pytest.param(
+            0, 0, [0, 0],
+            id="0 cat/dog years should convert into 0 human age."
+        ),
+        pytest.param(
+            14, 14, [0, 0],
+            id="14 cat/dog years should convert into 0 human age."
+        ),
+        pytest.param(
+            15, 15, [1, 1],
+            id="15 cat/dog years should convert into 1 human age."
+        ),
+        pytest.param(
+            23, 23, [1, 1],
+            id="15 cat/dog years should convert into 1 human age."
+        ),
+        pytest.param(
+            24, 24, [2, 2],
+            id="24 cat/dog years should convert into 2 human age."
+        ),
+        pytest.param(
+            27, 27, [2, 2],
+            id="27 cat/dog years should convert into 2 human age."
+        ),
+        pytest.param(
+            28, 28, [3, 2],
+            id="28 cat/dog years should convert into 3/2 human age."
+        )
+    ]
+)
```

Comment:
Add tests that pass large numbers, negative numbers, and different numbers to `cat_age` and `dog_age` as mentioned in the checklist. Besides that, declare another function to test against the raised error in case different invalid data types are passed to the function

Reply:
Done